docs/versioning.md

```diff
@@ -1,55 +1,46 @@
-# Release compatibility
+# Release versioning
 
 Modules are updated regularly. The changes adhere to [semantic versioning](https://semver.org/), with releases labeled as `{major}.{minor}.{patch}`. [Git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) are used to identify module versions.
 
-A major release level indicates a change that is not compatible with earlier versions. A minor release includes new capabilities and includes only compatible changes. A patch includes mostly bugs fixes that are compatible with an earlier version.
+Release levels apply to both modules and submodules. For example, a major version change to a submodule results in a major release for the whole module. However, changes to examples do not drive release levels, unless the example is published as a Deployable Architecture to the IBM Cloud.
```

Comment:
"unless the example is published as a Deployable Architecture to the IBM Cloud" -> I think we should graduate as example used as a deployable arch to a submodule to be fully consistent. It won't be possible for people to know what example are used without digging in the catalog json or in catalog ui.

Reply:
for SLZ, the only DA left in `examples` is the quickstart one. Should we move it into `patterns`?
For PowerVS and SAP, all of their DAs come from the `examples` folder. Should we advise that they move into a different folder?

Reply:
I'd move it out as one submodule under the vsi pattern